app/src/main/java/com/isc/hermes/database/IncidentsUploader.java

```diff
@@ -0,0 +1,95 @@
+package com.isc.hermes.database;
+
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * This class is for uploading incidents to the server.
+ */
+public class IncidentsUploader {
+    private final String API_URL = "https://api-rest-hermes.onrender.com/incidents";
```

Comment:
Since you are initializing this variable in the same structure, you might consider putting the API URL in `strings.xml `or declaring it as a class constant.

Reply:
This action can not be done because "R.string.incidents_api_url" is a resource identifier, which is an integer value that represents the unique identifier of a string resource. That's why when I try to  directly pass R.string.incidents_api_url to the URL constructor, it treats it as an integer and not the actual string value.